pkg/minishift/provisioner/minishift_provisioner.go

```diff
@@ -110,7 +110,7 @@ func (provisioner *MinishiftProvisioner) Provision(swarmOptions swarm.Options, a
 	swarmOptions.Env = engineOptions.Env
 
 	// set default storage driver for minishift
-	storageDriver, err := decideStorageDriver(provisioner, "devicemapper", engineOptions.StorageDriver)
+	storageDriver, err := decideStorageDriver(provisioner, "overlay2", engineOptions.StorageDriver)
```

Comment:
@praveenkumar  I guess for boot2docker or Minikube ISO we do not use minishift_provisioner, right?

Reply:
No the don't, minikube iso by default uses `overlay` driver and b2d by default using `aufs`. This code change only target for RHEL/CentOS to use `overlay`.